test/isArray.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { isArray } from "../src/index";
+
+describe("isString", function () {
+  it.each([2, true, "a"])(
```

Comment:
It would be better to put all types of javascript
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#description

`isString` too

Reply:
Thanks, I added all of the js types in here(2f2d020) 👍